packages/css/src/components/description-list/description-list.scss

```diff
@@ -25,19 +25,30 @@
 }
 
 @media screen and (min-width: $ams-breakpoint-medium) {
-  .ams-description-list {
+  .ams-description-list__group {
+    column-gap: var(--ams-description-list-column-gap);
+    display: grid;
+    grid-auto-flow: dense;
+    grid-column: span 2;
+  }
+
+  .ams-description-list,
+  .ams-description-list__group {
     grid-template-columns: auto 1fr;
   }
 
-  .ams-description-list--terms-width-sm {
+  .ams-description-list--terms-width-sm,
+  .ams-description-list--terms-width-sm .ams-description-list__group {
     grid-template-columns: 1fr 4fr;
   }
 
-  .ams-description-list--terms-width-md {
+  .ams-description-list--terms-width-md,
+  .ams-description-list--terms-width-md .ams-description-list__group {
     grid-template-columns: 1fr 2fr;
   }
 
-  .ams-description-list--terms-width-lg {
+  .ams-description-list--terms-width-lg,
+  .ams-description-list--terms-width-lg .ams-description-list__group {
     grid-template-columns: 1fr 1fr;
   }
 }
```

Comment:
A description wrapping over multiple lines will push the second and consecutive terms down. Can you check whether something like this could address that?

```scss
.ams-description-list__group {
  .ams-description-list__description {
    grid-row: 1 / -1;
  }
}
```

Reply:
This will also cause overlap when there are multiple descriptions.

Reply:
The grid rows need to be explicit for this to work. I would love to do this:

```css
@media screen and (min-width: $ams-breakpoint-medium) {
  .ams-description-list__group {
    dd:only-of-type {
      grid-row: 1 / -1;
    }
  }
}
```

But it won't work if the rows are set to auto.

Reply:
Proposed a solution with inline style to set rows.

Reply:
I've come to understand that the specification deliberately excludes implicit grids for negative indexes of grid item placement to prevent unpredictability and performance issues.

@dlnr @alimpens How about `grid-row: 1 / 999` as an alternative? This streches the cell to all three rows as well, doesn’t seem to cause any rendering issues in my Safari and Firefox and removes the need for inline styles.

Reply:
Not _very_ hacky.

Multiple terms sharing multiple declarations is an extreme edge case.

Inline styles can still be added and will override the 999 row span.

Let’s see which way Aram wants to go.